services/graylog/docker-compose.yml

```diff
@@ -108,10 +108,10 @@ services:
       resources:
         limits:
           cpus: "0.20"
-          memory: 256M
+          memory: 512M
```

Comment:
NOTE: at some point we should revisit a common strategy to review how and what we log! 
E.g. this issue since 2021 https://github.com/ITISFoundation/osparc-simcore/issues/2157